sdk/storage/azfile/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ### Breaking Changes
 
+* Changed type of SourceContentCRC64 header from uint64 to a generic interface implementation, so that it is considered 
```

Comment:
I think we mention the breaking changes seperatly. Just check if this is the correct format or not.